docs/running-on-yarn.md

```diff
@@ -113,9 +113,9 @@ Most of the configs are the same for Spark on YARN as for other deployment modes
 </tr>
 <tr>
  <td><code>spark.yarn.executor.memoryOverhead</code></td>
-  <td>executorMemory * 0.07, with minimum of 384 </td>
+  <td>executorMemory * 0.07, with minimum of 384 megabytes </td>
```

Comment:
Indeed, and should the other properties here get the same treatment, in docs and code?